include/xgboost/c_api.h

```diff
@@ -16,6 +16,10 @@
 #include <stdint.h>
 #endif
 
+#ifdef XGBOOST_USE_CUDF
+#include <cudf/types.h>
```

Comment:
Just a suggestion, is it possible to use opaque type in `c_api.h` ?  I have been trying to rewrite the CMake scripts and added a installation target,  bring in a external header will cause some trouble.

Reply:
I'm not sure what you mean. Could you clarify and provide an example?